lib/index.js

```diff
@@ -157,48 +157,59 @@ auth0GuardianJS.prototype.start = function start(callback) {
  */
 
 auth0GuardianJS.resume = function resume(options, transactionState, callback) {
-  var transactionTokenObject = jwtToken(transactionState.transactionToken);
-  var txId = transactionTokenObject.getDecoded().txid;
+  try {
+    var transactionTokenObject;
 
-  // create httpClient/socketClient
-  var httpClientInstance = object.get(options, 'dependencies.httpClient',
-    httpClient(transactionState.baseUrl, txId));
+    try {
+      transactionTokenObject = jwtToken(transactionState.transactionToken);
+    } catch (e) {
+      throw new errors.InvalidToken('Invalid transaction token');
```

Comment:
I think the nested try-blocks are an unnecessary complexity; couldn't this be handled in the same catch as the rest?  If there are more exceptions possible, just check for error name if it should be transformed?